scripts/windows/setup/Install-SecurityDaemon.ps1

```diff
@@ -1,32 +1,41 @@
+new-module -name IoTEdge -scriptblock {
```

Comment:
Use consistent casing `New-Module` (the rest of the file uses Pascal case for powershell commands)

Reply:
Done